### PR TITLE
Add sepolia migration updates

### PR DESCRIPTION
**Pull request by AlbionaHoti**
# What :computer: 
- More fixes regarding the sepolia migration

# Evidence :camera:
Include screenshots, screen recordings, or `console` output here demonstrating that your changes work as intended

# Notes :memo:
* Any notes/thoughts that the reviewers should know prior to reviewing the code?

### CODE REVIEW

File: docs/dev/tutorials/aa-daily-spend-limit.md [hunk 1]
```diff
@@ -765,7 +765,7 @@ contract AAFactory {
 yarn hardhat compile
 ```
 
-2. Create a file `deploy/deployFactoryAccount.ts` and copy/paste the code below, replacing `<DEPLOYER_PRIVATE_KEY>` with your own.
+2. Create a file `deploy/deployFactoryAccount.ts` and copy/paste the code below, don't forget to add `DEPLOYER_PRIVATE_KEY` in your `.env` file.
```
Comment (dutterbutter) [inline docs/dev/tutorials/aa-daily-spend-limit.md:768]:
```suggestion
2. Create a file named `deploy/deployFactoryAccount.ts`. Then, copy and paste the following code into it. Remember to add your `DEPLOYER_PRIVATE_KEY` to the .env file.
```

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ head:
 
 # USDC paymaster tutorial with API3 dAPIs
 
+:::Note
+The team is migrating to the Sepolia testnet, and for this tutorial, we're updating the contract addresses for `ETHUSDdAPI` and `USDCUSDdAPI` with the API3 team. Only then will the tutorial be functional in the zkSync Sepolia testnet. For now, it's available only with the Goerli testnet.
+:::
```
Comment (dutterbutter) [inline docs/dev/tutorials/api3-usd-paymaster-tutorial.md:12]: I'd rather create a tip warning that specifies:
```
This tutorial is currently operational exclusively on the Goerli testnet. Stay tuned for upcoming support for the Sepolia network.
```

Much cleaner imo.

**Review by dutterbutter — CHANGES_REQUESTED** [review]
Couple changes.

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md (hunk 2 above)
Comment (AlbionaHoti) [reply docs/dev/tutorials/api3-usd-paymaster-tutorial.md:12]: Yes it is cleaner, thanks for suggesting!

**Review by dutterbutter — APPROVED** [review]

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**